libs/auth/scripts/configure_java_card.ts

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable no-console */
+import { Buffer } from 'buffer';
+import { spawnSync } from 'child_process';
+import { assert, sleep } from '@votingworks/basics';
+import { Byte } from '@votingworks/types';
+
+import { CommandApdu, constructTlv } from '../src/apdu';
+import {
+  CARD_VX_ADMIN_CERT,
+  CARD_VX_CERT,
+  DEFAULT_PIN,
+  GENERIC_STORAGE_SPACE,
+  JavaCard,
+  MAX_INVALID_PIN_ATTEMPTS,
+  OPEN_FIPS_201_AID,
+  PUK,
+  VX_ADMIN_CERT_AUTHORITY_CERT,
+} from '../src/java_card';
+import {
+  construct8BytePinBuffer,
+  CRYPTOGRAPHIC_ALGORITHM_IDENTIFIER,
+} from '../src/piv';
+
+const APPLET_PATH = 'applets/OpenFIPS201-v1.10.2-with-vx-mods.cap';
```

Comment:
Though I've committed the applet as part of this PR, I'll still be putting up a separate PR to review our modifications to OpenFIPS201. And will update here if necessary. The CAP file included in this PR was built using the steps laid out here and does have the modification we want.

I could see a case for building the applet as part of vxsuite-complete-system, but we should almost never have to modify it. Putting it in vxsuite pre-built also makes card configuration for development easier.

We can also easily revisit this.

Reply:
until we've got the new build approach from @amcmanus I think it makes sense to keep it here.